docs/usage/automerge-configuration.md

```diff
@@ -138,3 +138,22 @@ These approval helper apps are only available for GitHub.
 ### Codeowners
 
 Depending on the platform, having a `CODEOWNERS` file could block automerging, because it means a code owner must review the PR.
+
+### Disabling automerge in select repos, rest of org should automerge
```

Comment:
I'm not sure if this `h3` title is the correct summary of the section below, it's a bit long right now.

If you have any ideas, please suggest them!

Reply:
Maybe `### Overriding global automerge`

Reply:
```suggestion
### Overriding global automerge
```

Let's go with that for now! 😄